### ISSUE TITLE
Editor: WordPress Reader view showing 2 different post views, 1 without line breaks for posts created in the Classic Editor/Classic Block?

### ISSUE THREAD

**Issue by inaikem** [Automattic/wp-calypso#46710]
Visit a post created using the Classic Editor (no P tags) in the WordPress Reader. 

Depending on the type of link you follow, you will be shown a "normal" view of the post, or a view without linebreaks, giving the appearance of "bunched up" text.

See the example below:

&nbsp;
> ![](
https://d.pr/i/QxkKfi+)
[Can't see the image? Click this direct link.](
https://d.pr/i/QxkKfi)

&nbsp;
This is the same post.

- Left URL (structure): https://wordpress.com/read/blogs/123/posts/456
- Right URL (structure): https://wordpress.com/read/feeds/789/posts/012

Is post content being output differently depending on if it is ingested/output at `/read/blogs/` vs `/read/feeds/`?

**Comment by pablinos** [comment]
I'm not sure what was causing this, but it appears to have fixed itself. With the difference in view between staging and production, I suspect there was some caching at play. What caused it to lose the formatting in the first place will be tricky to determine. Let's monitor things and see if we get any more instances of this.

In the meantime, I've opened an issue to track the problem we're seeing with featured images automattic/dotcom-manage#290

**Comment by cathymcbride** [comment]
If this is fixed, and the featured images problem is being tracked elsewhere, I'm closing this one. Please feel free to re-open if it comes up again.

**Comment by rinazrina** [comment]
I'm reopening this thread as I got a similar report on #25360839-hc - detailed notes on #3459449-zen

Feed URLs of the post in question:
- https://wordpress.com/read/blogs/177978112/posts/1566
- https://wordpress.com/read/feeds/107078562/posts/2992857371

User is using Classic block for this post.

**Comment by cathymcbride** [comment]
@rinazrina are you able to recreate this yourself?

**Comment by rinazrina** [comment]
No, I can't reproduce this. I created a test post using the Classic block but no line breaks issue on both the /feeds and /blogs URLs. 

- https://wordpress.com/read/feeds/107947238/posts/3002529367
- https://wordpress.com/read/blogs/167335396/posts/686

However, I just checked again and see the issue has been fixed itself on the user's site that I reported above. So it's safe to assume this is a cache issue, yes?
